### PR TITLE
Icon Tweaks

### PR DESCRIPTION
**Pull request by xtrm-en**
## Description
This PR brings the following features:
- Removes the Mod Card for the `Forge Mod Loader` (modid: `FML`) mod, as it's not useful to display
- Adds a Mod Icon for the `MinecraftForge` (modid: `Forge`) mod
- ModCard icons now render up to a `56x56` resolution, instead of the previous `48x48`.
  - Subsequently, the rendering code for ModCard icons has been moved

## Checklist
- [x] I made a clear description of what was changed
- [x] I stated why these changes were necessary
- [x] I updated documentation or said what needs to be updated
- [x] I made sure these changes are backwards compatible
- [x] This pull request is for one feature/bug fix

### CODE REVIEW

**Review by Wyvest — CHANGES_REQUESTED** [review]
fine overall but can you implement rectangle icons and where's the forge icon

**Comment by xtrm-en** [comment]
rectangle icons would be possible but kinda bad design-wise imo

> and where's the forge icon

[wdym](https://github.com/Polyfrost/OneConfig/pull/229/commits/23e46da213997f077c2703419fb393d3ea7a1f67)

**Comment by Wyvest** [comment]
> rectangle icons would be possible but kinda bad design-wise imo
> 

mmm but like our fallback is literally text (which acts like a rectangle)

> 
> > and where's the forge icon
> 
> 
> 
> [wdym](https://github.com/Polyfrost/OneConfig/pull/229/commits/23e46da213997f077c2703419fb393d3ea7a1f67)

i love github mobile

**Comment by xtrm-en** [comment]
then ways of implementing it would be to either:
- look at the aspect ratio of the provided icon and use that to determine the width from the fixed height
  - idk if that would work for svgs
- let the user provide the icon dimentions  
  - they're gonna fuck it up (probably)

**Comment by Wyvest** [comment]
> then ways of implementing it would be to either:
> 
> - look at the aspect ratio of the provided icon and use that to determine the width from the fixed height
> 
>   - idk if that would work for svgs
> 
> - let the user provide the icon dimentions  
> 
>   - they're gonna fuck it up

let the users provide it, it's not our fault if it's stretched as hell

**Comment by xtrm-en** [comment]
should we postpone rectangle icons to config rewrite?

**Comment by Wyvest** [comment]
why though

**Comment by xtrm-en** [comment]
because the `Mod` constructor is already cluttered enough as-is, it's not an urgent feature, and I'd like to add a bit more than just dimensions

**Comment by Wyvest** [comment]
i mean slightly urgent we have devs waiting for this

**Comment by xtrm-en** [comment]
oh? well alright then

File: versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
 import cc.polyfrost.oneconfig.utils.Notifications;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.gui.GuiScreen;
+import net.minecraftforge.fml.common.ModMetadata;
```
Comment (Wyvest) [inline versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java:53]: put this in the forge comment block

**Review by Wyvest — CHANGES_REQUESTED** [review]

**Review by Wyvest — APPROVED** [review]
yes

**Pull request merged**